examples/drive_cycle_comparison.py

```diff
@@ -0,0 +1,103 @@
+import pybamm
```

Comment:
would be good to add a few more comments in this example explaining that you are comparing solving a model in liiopack and in pybamm